apps/judicial-system/web/src/routes/Prosecutor/Indictments/CaseFiles/CaseFiles.tsx

```diff
@@ -123,6 +123,23 @@ const CaseFiles: React.FC = () => {
             onRetry={handleRetry}
           />
         </Box>
+        <Box component="section" marginBottom={10}>
```

Comment:
According to the screenshot you posted, I think you need to change the marginBottom on the section above this one.